src/connectors/queue/publication.ts

```diff
@@ -361,6 +337,11 @@ class PublicationQueue extends BaseQueue {
         )
       }
 
+      await this.articleService.baseUpdate(article.id, {
```

Comment:
shall we move [update `draft.publishState` to `published`](https://github.com/thematters/matters-server/blob/b95b0de1c1d79986a7d9ef1e72e6797f0c596a9b/src/connectors/queue/publication.ts#L141) to here?